README.md

```diff
@@ -1,4 +1,4 @@
-# ert [![Build Status](https://travis-ci.org/equinor/ert.svg?branch=master)](https://travis-ci.org/equinor/ert)
+# ert [![Build Status](https://travis-ci.org/equinor/ert.svg?branch=master)](https://travis-ci.org/equinor/ert)[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
```

Comment:
Very nitpick, but maybe include a space?
```suggestion
# ert [![Build Status](https://travis-ci.org/equinor/ert.svg?branch=master)](https://travis-ci.org/equinor/ert) [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
```

Reply:
If there is a PR that should accept style comments, it better be this one 😂 I was just about to check how it rendered... Also, the `travis` badge is outdated, so I'm creating a third PR 🚢